package.json

```diff
@@ -50,6 +50,8 @@
     "d3": "3.5.16",
     "date-fns": "1.30.1",
     "dygraphs": "2.1.0",
+    "fast-deep-equal": "2.0.1",
```

Comment:
We already bundle lodash.isequal... Should we just reuse that or maybe migrate other callers?

Reply:
@DanielMSchmidt you might want to follow up on this 👼

Reply:
Followup: https://github.com/dcos/dcos-ui/pull/3795